README.md

```diff
@@ -0,0 +1,255 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="logo.webp" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Anime Facts</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [🚀 Live Video](#live-video)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 To Do List <a name="about-project"></a>
+
+**Anime Facts** is a web application that allows users to view facts and details about their favorite anime shows.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org">React JS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  No server-side technologies were used for this project.
+</details>
+
+<details>
+<summary>Database</summary>
+  No database was used for this project.
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **View Anime Facts**
+- **View Anime Details**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo](https://silver-crisp-2f216f.netlify.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE VIDEO -->
+
+## 🚀 Live Video <a name="live-video"></a>
+
+[Live Video](https://www.loom.com/share/08e77e4206e542e8b92b4405021b9caa?sid=620f7dbb-3347-4b71-973a-923c4f523d27)
```

Comment:
- [ ] Great job @10-menachi There are just a few changes I will suggest you make and you'll be on your way to the next milestone :wink: 
- [ ] Great job adding the Presentation video :clap: However, I noticed that the video doesn't contain a highlight of a piece of the code that you find interesting or are proud of making. :+1: 
- [ ] Also, the design you were given was a mobile design but in your video, you didn't show the mobile view you only showed the desktop view. So kindly redo the video and ensure that it is in mobile view and also contains the earlier comment :wink: